### PR TITLE
Bugfix for issue #139 brace completion should be disabled in comments 

### PR DESCRIPTION
**Pull request by HuanhuanSunMSFT**
Add a method to determine if the typed left brace is in comments region, only trigger brace completion when the method returns true.
@AndreSayreMSFT @EricMSFT

### CODE REVIEW

File: PowerShellTools/Intellisense/BraceCompletionController.cs [hunk 1]
```diff
@@ -139,16 +145,27 @@ public int Exec(ref Guid pguidCmdGroup, uint nCmdID, uint nCmdexecopt, IntPtr pv
                     break;
             }
 
-            return NextCommandHandler.Exec(ref pguidCmdGroup, nCmdID, nCmdexecopt, pvaIn, pvaOut);
+            return VSConstants.S_FALSE;
         }
 
-        public int QueryStatus(ref Guid pguidCmdGroup, uint cCmds, OLECMD[] prgCmds, IntPtr pCmdText)
+        private bool IsInCommentArea()
```
Comment (ASayre) [inline PowerShellTools/Intellisense/BraceCompletionController.cs:153]: Inperson convo re: testability

**Comment by ASayre** [comment]
Merge conflicts!

**Comment by HuanhuanSunMSFT** [comment]
Fixes #139

**Comment by ASayre** [comment]
Looks good!

**Pull request merged**